### PR TITLE
Booster fleet unit

### PR DESCRIPTION
**Pull request by phylake**
/cc @ts3ng @eadasiak @lrouquette

### CODE REVIEW

File: v3/fleet-local/control/booster-metrics-producer@.service [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+User=core
+Restart=on-failure
+
+ExecStartPre=/usr/bin/sh -c "/usr/bin/docker pull $(etcdctl get /images/booster/metrics-producer)"
+ExecStartPre=-/usr/bin/docker kill booster
+ExecStartPre=-/usr/bin/docker rm booster
+ExecStart=/usr/bin/sh -c "/usr/bin/docker run \
+  --name booster \
+  --net=host \
+  -e AWS_DEFAULT_REGION=us-east-1 \
```
Comment (phylake) [inline v3/fleet-local/control/booster-metrics-producer@.service:16]: I'm not in love with this. Is there a good alternative or precedent for this kind of thing?

Comment (chr0n1x) [inline v3/fleet-local/control/booster-metrics-producer@.service:16]: maybe source this data from the metadata service? Something like [this](https://github.com/adobe-platform/mesos-systemd/blob/master/v3/setup/fleet.sh#L13)?

File: v3/setup/images-control.sh [hunk 2]
```diff
@@ -12,12 +12,13 @@ if [ "$(etcdctl get /bootstrap.service/images-control-bootstrapped)" == "true" ]
...
-etcdctl set /images/mesos-master 	"mesosphere/mesos-master:0.27.0-0.2.190.ubuntu1404"
-etcdctl set /images/zk-exhibitor 	"behance/docker-zk-exhibitor:latest"
-etcdctl set /images/cfn-signal   	"behance/docker-cfn-bootstrap:latest"
+etcdctl set /images/chronos                   "mesosphere/chronos:chronos-2.4.0-0.1.20150828104228.ubuntu1404-mesos-0.27.0-0.2.190.ubuntu1404"
+etcdctl set /images/flight-director           "behance/flight-director:latest"
+etcdctl set /images/marathon                  "mesosphere/marathon:v0.15.1"
+etcdctl set /images/mesos-master              "mesosphere/mesos-master:0.27.0-0.2.190.ubuntu1404"
+etcdctl set /images/zk-exhibitor              "behance/docker-zk-exhibitor:latest"
+etcdctl set /images/cfn-signal                "behance/docker-cfn-bootstrap:latest"
+etcdctl set /images/booster/metrics-producer  "behance/booster:fdcloudwatch-latest"
```
Comment (matthewdfuller) [inline v3/setup/images-control.sh:21]: Is this a typo? `behance/booster:fdcloudwatch-latest`? Typically the tag name is `:latest` not `-latest`.

EDIT: nevermind, I see the image is `booster` and the tag is `fdcloudwatch-latest`.

**Comment by matthewdfuller** [comment]
Does this PR enable Booster to send metrics to CloudWatch? If so, is there an equivalent infrastructure PR that gives the instances permissions to put metrics, create log groups, etc?

File: v3/fleet-local/control/booster-metrics-producer@.service (hunk 1 above)
Comment (eadasiak) [inline v3/fleet-local/control/booster-metrics-producer@.service:16]: Like this?

-e AWS_DEFAULT_REGION=$(curl -s http://169.254.169.254/latest/dynamic/instance-identity/document|jq .region)

Comment (phylake) [inline v3/fleet-local/control/booster-metrics-producer@.service:16]: Sure i was hoping there was an etcd value already set or something. Thanks all I'll update this

**Comment by phylake** [comment]
@matthewdfuller No, that's a good point though. This isn't going to take another service down though if permissions aren't right. Worst case we produce lots of log noise with errors.

@ts3ng is the infrastructure bit something you can work on? I'm not sure what permissions are needed

**Comment by ts3ng** [comment]
@phylake  i have a ticket here to add the policy.
https://github.com/adobe-community/issues/issues/9429

**Comment by ts3ng** [comment]
@phylake i've created a PR to update the control tier w/ a booster policy.  @matthewdfuller  please take a look at https://github.com/adobe-platform/infrastructure/pull/96

**Comment by ts3ng** [comment]
@phylake  we shoudl also load the all env variables as etcd values so they can be configurable without reloading the service. ie /booster/BOOSTER_CLOUDWATCH_FREQUENCY

**Comment by phylake** [comment]
Will re-open in ethos-systemd if needed

**Pull request closed without merging**